src/FindApprenticeshipTraining/SFA.DAS.FindApprenticeshipTraining/InnerApi/Requests/GetProvidersByCourseIdRequest.cs

```diff
@@ -1,19 +1,75 @@
+using SFA.DAS.FindApprenticeshipTraining.Application.Courses.Queries.GetCourseProviders;
 using SFA.DAS.SharedOuterApi.Interfaces;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace SFA.DAS.FindApprenticeshipTraining.InnerApi.Requests;
 
 public class GetProvidersByCourseIdRequest : IGetApiRequest
 {
-    private readonly double? _latitude;
-    private readonly double? _longitude;
-    private readonly int _courseId;
+    public required int CourseId;
+    public ProviderOrderBy? OrderBy;
+    public decimal? Distance;
+    public decimal? Latitude;
+    public decimal? Longitude;
+    public List<DeliveryMode?> DeliveryModes;
+    public List<ProviderRating?> EmployerProviderRatings;
+    public List<ProviderRating?> ApprenticeProviderRatings;
+    public List<QarRating?> Qar;
+    public int? Page;
+    public int? PageSize;
```

Comment:
All these should be public properties i.e. `public int CourseId {get; init;}` use `init` where it is required